napari/layers/points/points.py

```diff
@@ -309,20 +315,21 @@ def symbol(self, symbol: Union[str, Symbol]) -> None:
 
     @property
     def sizes(self) -> Union[int, float, np.ndarray, list]:
-        """(N, D) array: sizes of all N points in D dimensions."""
+        """(N,) array: sizes of all N points."""
         return self._sizes
 
     @sizes.setter
     def sizes(self, size: Union[int, float, np.ndarray, list]) -> None:
-        try:
-            self._sizes = np.broadcast_to(size, self.data.shape).copy()
-        except:
-            try:
```

Comment:
Note @jni if we go with the strict broadcasting approach I will close this PR and make a new one that only drops this extra try and changes the tests / examples appropriately. Does this sound good to you?